repo/mk/js.deps.npm.mk

```diff
@@ -79,3 +76,30 @@ deps-npm-prod:
 		$(NPM) update --no-save --production; \
 	}
 	$(NPM) list --depth=0 || $(MAKE) deps-npm-unmet-peer
+
+
+.PHONY: deps-npm-audit
+deps-npm-audit:
+	$(ECHO_DO) "Auditing package.json..."
+	$(GIT) ls-files --error-unmatch "package-lock.json" 2>/dev/null || { \
+		$(ECHO_INFO) "Creating intermediary package-lock.json needed by 'npm audit'..."; \
+		$(RM) package-lock.json; \
+		$(NPM) install --no-audit --package-lock-only 2>/dev/null; \
+	}
+	$(NPM) audit || { \
```

Comment:
made a discovery: apparently you cannot audit a "private" package i.e. if `package.json` has `private: true` (which was supposed to be a good way to stop someone from erroneously publishing the package to an npm registry...)

might be a BLOCKER to move forward on this, and might need to back down to having this audit as optional